Meadowlark-js/backends/meadowlark-mongodb-backend/src/repository/authorization/CreateClient.ts

```diff
@@ -0,0 +1,50 @@
+// SPDX-License-Identifier: Apache-2.0
+// Licensed to the Ed-Fi Alliance under one or more agreements.
+// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+// See the LICENSE and NOTICES files in the project root for more information.
+
+import { Collection, ClientSession, MongoClient } from 'mongodb';
+import { Logger } from '@edfi/meadowlark-core';
+import { CreateClientRequest, CreateClientResult } from '@edfi/meadowlark-authz-server';
+import { AuthorizationClient, authorizationClientFrom } from '../../model/AuthorizationClient';
+import { asUpsert, getAuthorizationCollection } from '../Db';
+
+export async function createAuthorizationClient(
```

Comment:
Putting this in a separate "authorization" folder under "repository" while we consider single backend vs. separate. If we go single, thinking move the document stuff into a "document" folder.